test/run

```diff
@@ -45,34 +45,50 @@ run_s2i_build() {
   s2i build ${s2i_args} file://${test_dir}/test-app ${IMAGE_NAME} ${IMAGE_NAME}-testapp
 }
 
+run_s2i_build_proxy() {
+    s2i build ${s2i_args} -e HTTP_PROXY=$1 -e http_proxy=$1 -e HTTPS_PROXY=$2 -e https_proxy=$2 file://${test_dir}/test-hw ${IMAGE_NAME} ${IMAGE_NAME}-testhw
+}
+
 prepare() {
   if ! image_exists ${IMAGE_NAME}; then
     echo "ERROR: The image ${IMAGE_NAME} must exist before this script is executed."
     exit 1
   fi
   # TODO: STI build require the application is a valid 'GIT' repository, we
   # should remove this restriction in the future when a file:// is used.
-  pushd ${test_dir}/test-app >/dev/null
+  if [[ -z $1 ]]; then
```

Comment:
Would be better if the contents of the argument were used to change to the directory specified instead. Will be useful if there are any additional tests added in the future.